src/pyramid/config/routes.py

```diff
@@ -220,7 +220,8 @@ def add_route(
         header
 
           This argument represents an HTTP header name or a header
-          name/value pair.  If the argument contains a ``:`` (colon),
+          name/value pair, or a sequence of them.
+          If the argument contains a ``:`` (colon),
```

Comment:
Suggestion: 
```suggestion
          If one of the arguments contains a ``:`` (colon),
```
(just to be explicit that it is evaluated for each element, not for every elements at once)

Reply:
What do you think of this:
```suggestion
          If a value contains a ``:`` (colon),
```
to avoid clunky «if the argument or one of the arguments»

Reply:
Perfect 👌

Reply:
Ah, that doesn’t work for the second instance below that already uses «value» instead of «parameters».
I think I have to sit back and rewrite the whole four lines here to make it read better.

Reply:
"If any ____"?

Reply:
I found that my issue was the repeated switching between «if this is a header name only» and «if the value contains a colon», so I grouped all info about name-only values then all info about name-with-value-regex values to make it read better.